.openpublishing.redirection.json

```diff
@@ -294,6 +294,10 @@
         "redirect_url": "/visualstudio/ide/visual-studio-ide"
     },
     {
+        "source_path": "docs/visual-basic/getting-started/breaking-changes-in-visual-studio-2015.md", 
```

Comment:
@mairaw Since we removed the 2013 version in the csharp docs and the 2015 version is currently in the VB area (and has VB is in the title), I'll await your instructions on where you want this.